packages/nestjs-custom-decorators/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "nestjs-custom-decorators-decorators",
+  "version": "1.0.0",
+  "description": "custom nest decorators for nestjs-custom-decorators",
+  "keywords": [
+    "nest",
+    "decorator",
+    "jwt"
+  ],
+  "author": "meriamBenSassi <meriam.bensassi@algoan.com>",
+  "homepage": "https://github.com/algoan/nestjs-components/tree/master/packages/nestjs-custom-decorators-decorators#readme",
+  "license": "ISC",
+  "main": "lib/nestjs-custom-decorators-decorators.js",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/algoan/nestjs-components.git"
+  },
+  "scripts": {
+    "test": "NODE_ENV=test mocha --reporter spec --exit --require ts-node/register ./test/**/*.test.ts"
+  },
+  "bugs": {
+    "url": "https://github.com/algoan/nestjs-components/issues"
+  },
+  "dependencies": {
+    "@nestjs/common": "^7.0.7",
+    "jwt-decode": "^2.2.0",
+    "reflect-metadata": "^0.1.13"
```

@nestjs/common and reflect-metadata should also be in peer dependencies and dev dependencies

```suggestion
    "jwt-decode": "^2.2.0",
```